staging/src/k8s.io/apiextensions-apiserver/test/integration/main_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 The Kubernetes Authors.
+Copyright 2020 Authors of Arktos.
```

Comment:
?

Reply:
It is a file created by me and accidentally checked in the CRD PR. Here the copyright statement is fixed.